includes/class-wc-stripe-helper.php

```diff
@@ -726,12 +726,21 @@ public static function filter_payment_methods_with_capabilities( $payment_method
 			return [];
 		}
 
+		// Return all payment methods if in test mode.
+		if ( $testmode ) {
+			return $payment_method_ids;
+		}
```

Comment:
Oh good change. 👍 Skipping the whole process in test mode makes so much more sense 😂